helper.py

```diff
@@ -1,41 +1,119 @@
-#!/usr/bin/env python3
+'''
+IMPORTANT NOTE!!
 
-# This file may be obsolete - need to check if scaling in shephard is laggy
+=> The following will ONLY run on Python 3.7 to 3.9
+=> Install websockets version 11.0.3 via pip whl (do not use v12)
+=> Install asynio 0.2.0 via pip whl
 
-import os
-import time
+I will also update sheep to consume the websocket at the other end.
+'''
 
+import asyncio
+import aionotify
+from websockets.server import serve
+import websockets
+import time
+import base64
+import os
+from io import BytesIO
 from PIL import Image
 
 
-static_path = 'shepherd/static/'
+CONNECTIONS = set()
+
+#The following sets up the asynchronous waiting for file change
+watcher = aionotify.Watcher()
+
+#TODO - Change the input_file to the location of the image the camera takes!
+static_path = '/home/pi/shepherd/shepherd/static/'
 input_file = static_path + "full_image.jpg"
```

Comment:
This needs to be updated to image.jpg, and there needs to be some kind change to the output file to make it fit with your brains, otherwise you may start detecting your own overwrites.

Reply:
Yep will do. Do we really need to save the image to disk? Was thinking we could just fire it over the websocket and move on. Would also be lighter on resources.